src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx

```diff
@@ -36,9 +36,12 @@ export default class MutationAssessorColumnFormatter
         };
     }
 
-    public static getSortValue(d:Mutation[]):(number|undefined)[]
+    public static getSortValue(d:Mutation[]):(number|null)[]
     {
-        const score = MutationAssessorColumnFormatter.getData(d).score;
+        let score:number|undefined|null = MutationAssessorColumnFormatter.getData(d).score;
+        if (score === undefined) {
```

Comment:
we should try to comment stuff that expresses some obscure business logic.  Like, why can store be undefined. E.g. "sometimes data is missing"